src/mock-api-test/java/com.smartsheet.api/sdk_test/HelperFunctions.java

```diff
@@ -0,0 +1,19 @@
+package com.smartsheet.api.sdk_test;
+
+import com.smartsheet.api.Smartsheet;
+import com.smartsheet.api.SmartsheetBuilder;
+import com.smartsheet.api.oauth.Token;
+
+public class HelperFunctions {
+	public static Smartsheet SetupClient(String apiScenario){
+		Token token = new Token();
+		token.setAccessToken("aaaaaaaaaaaaaaaaaaaaaaaaaa");
```

Comment:
Is there a reason you have to create a token and then just return what you passed in?

Reply:
No, i was just fallowing the example from other tests and/or the documents. I can just put the token string in the smartsheetbuilder.